### ISSUE TITLE
I

### ISSUE THREAD

**Issue by AlhassanMousa** [tr3v3r/react-native-esc-pos-printer#98]

**Comment by Rehankhalil462** [comment]
@AlhassanMousa  Yes. You are right. I checked but arabic is not in available languages. 
You can refer this for an alternative workaround . 
https://github.com/tr3v3r/react-native-esc-pos-printer/issues/54#issuecomment-1104241671

**Comment by mi3lix9** [comment]
Is there a plan to fix Arabic?

**Comment by tr3v3r** [comment]
> Is there a plan to fix Arabic?

Is should be possible natively but you have to play with encoding yourself.

I'd suggest printing from React Native View. In this case you can print whatever you want 

https://github.com/tr3v3r/react-native-esc-pos-printer/blob/main/docs/QUICK_START.md
